src/index.html

```diff
@@ -11,6 +11,35 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      action="/"
+      method="get"
+      data-qa="big"
+    >
+      <div class="input-wrapper input-wrapperBig">
```

Comment:
So, if you use a modifier for an element, you have to name it as `input-wrapper--big`. 
![image](https://user-images.githubusercontent.com/25587299/208320250-cd1ba999-8eb7-40aa-9083-a9835fed2fc8.png)